R/stan_module.R

```diff
@@ -1,44 +1,55 @@
 #' StanAll class object
-#' @slot functions Character, the functions part of a stan model
-#' @slot data Character, the data part of a stan model
-#' @slot parameters Character, the parameters part of a stan model
-#' @slot transformed_parameters Character, the transformed_parameters part of a stan model
-#' @slot model Character, the model part of a stan model
-#' @slot prior List, the priors part of a stan model
-#' @slot generated_quantities Character, the generated_quantities part of a stan model
+#' @slot functions Character, the functions part of a stan model.
+#' @slot data Character, the data part of a stan model.
+#' @slot parameters Character, the parameters part of a stan model.
+#' @slot transformed_parameters Character, the transformed_parameters part of a stan model.
+#' @slot model Character, the model part of a stan model.
+#' @slot priors List, the priors part of a stan model.
+#' @slot generated_quantities Character, the generated_quantities part of a stan model.
 #' @slot includes Character
 #' @slot inits List with the initial values of the stan model.
 #' @exportClass StanModule
 StanModule <- setClass(
-    "StanModule",
-    representation(
-        functions = "character",
-        data = "character",
-        parameters = "character",
-        transformed_parameters = "character",
-        prior = "list",
-        generated_quantities = "character",
-        inits = "list"
-    )
+  "StanModule",
+  representation(
+    functions = "character",
+    data = "character",
+    parameters = "character",
+    transformed_parameters = "character",
+    priors = "list",
+    generated_quantities = "character",
+    inits = "list"
+  )
 )
 
 
 
-#' source_stan_part exports a .stan file as a character
-#' @param filename The name of the .stan file in the directory of the package.
+#' read_stan_part returns stan code as a character.
+#' @param file Character, either the absolute path of a stan file, or the name of the stan file in the package directory or the stan code as a string.
 #' @export
-source_stan_part <- function(filename) {
-    if (file.exists(filename)) {
-        absolute_filename <- filename
+read_stan_part <- function(file) {
+    if (file.exists(file)) {
+        absolute_filename <- file
```

Comment:
I think this line can be deleted and just pass `file` to the `readLines` function